data/json/items/book/cooking.json

```diff
@@ -357,7 +360,7 @@
     "id": "recipe_raptor",
     "type": "BOOK",
     "name": "PE070 \"Raptor\": Proposal",
-    "name_plural": "PE070 \"Raptor\": Proposal",
+    "name_plural": "PE070 \"Raptor\": Papers",
```

Comment:
This makes it appear that the name of the proposal changes if you're carrying two of it

Reply:
```suggestion
    "name_plural": "PE070 \"Raptor\": Proposals",
```